__tests__/components/Link.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { Link } from '../../src/components';
+
+describe('Link', () => {
+  it('renders a default Link', () => {
+    const tree = renderer.create(<Link description={'description'} url={'url'} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
does the Link component have other cases, that could be added to test?

Reply:
yes, thanks . done in 9d3b4be